src/test/java/com/andersen/tr/task11/CreateCarTest.java

```diff
@@ -0,0 +1,88 @@
+package com.andersen.tr.task11;
+
+import com.andersen.tr.model.Car;
+import com.andersen.tr.model.CarType;
+import com.andersen.tr.model.Person;
+import com.andersen.tr.model.PersonStatus;
+import com.andersen.tr.repository.CarRepository;
+import com.andersen.tr.service.impl.CarService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateCarTest {
+    private CarService carService;
+
+    @Mock
+    private CarRepository carRepository;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
this method returns an instance of AutoClosable which can be used to close the resource after the test

simplify the way of mock creation and injection (use @ExtendWith(MockitoExtension.class) as a class annotation, remove setUp method, add @InjectMocks for CarService)